utilities/unittests/test_pytest_utils.py

```
@@ -1264,12 +1264,46 @@ def test_get_cnv_version_explorer_url_missing_env(self):
+            "install": False,
+            "upgrade": None,
+            "upgrade_custom": None,
+        }.get(option)
 
         result = get_cnv_version_explorer_url(mock_config)
 
         assert result is None
 
+    @patch("utilities.pytest_utils.os.environ", {"CNV_VERSION_EXPLORER_URL": "https://version-explorer.com"})
+    @patch("utilities.pytest_utils.LOGGER")
+    def test_get_cnv_version_explorer_url_upgrade_custom_cnv(self, mock_logger):
```

Ruff (0.15.21)

[warning] Unused method argument: `mock_logger`

(ARG002)